actionview/lib/action_view/helpers/tags/collection_check_boxes.rb

```diff
@@ -9,29 +9,13 @@ class CollectionCheckBoxes < Base # :nodoc:
         class CheckBoxBuilder < Builder # :nodoc:
           def check_box(extra_html_options={})
             html_options = extra_html_options.merge(@input_html_options)
+            html_options[:multiple] = true
             @template_object.check_box(@object_name, @method_name, html_options, @value, nil)
           end
         end
 
         def render(&block)
-          rendered_collection = render_collection do |item, value, text, default_html_options|
-            default_html_options[:multiple] = true
```

Comment:
@rafaelfranca I moved the multiple options to the [`check_box`](https://github.com/rails/rails/pull/18303/files#diff-ad5a6748d86270f717368d13026b475bR12) it is covered by the tests, if I remove this line I got some broken tests